### DISCUSSION TITLE
x-ms-webview control is not working in Store commerce

### DISCUSSION THREAD

**Discussion by Ponrathi-blip** [microsoft/Dynamics365Commerce.Solutions#258] [Q&A]
We have used x-ms-webview html control element to bind the webpage content in POS Dual Display, which is not working after Store commerce migration and it was working earlier in Retail SDK POS. So I have replaced the control with Iframe element but it has security restrictions and not loading all the website urls (ex: https://www.adidas.co.in) which is not in same origin. Kindly someone can help me to overcome this or suggest any alternate control. 

<x-ms-webview id="webview" src="http://www.bing.com"></x-ms-webview>

**Comment by Ponrathi-blip** [comment]
@madyke I have seen your response for Dual Display query. If you have any idea on this, please help. Since I'm also using Dual Display in Storecommerce App which has images, video and webpage. Now I have issue in loading Webpage content.

**Comment by zhangguanghuib** [comment]
Hello @Ponrathi-blip    I face the same issue, can you please try this:

<iframe src="https://www.bing.com" width="100%" height="600px" frameborder="0"></iframe>

to see if it can work

**Reply by Ponrathi-blip** [reply]
Hi @zhangguanghuib I've already tried using Bing.com, and it seems to be working OK, but I need to display any website urls that the customer provides. Since most websites have the option to deny x-frame, for instance, if they access https://www.pantaloons.com, it should load inside the iframe; however, this is not happening.

**Reply by zhangguanghuib** [reply]
Then the issue is not comes from iframe or x-ms-webview, the site itself https://www.pantaloons.com deny being embed into iframe,  you should change the site setting to allow that
